### PR TITLE
3DS2 Template and Controller

### PR DESCRIPTION
**Pull request by maassenbas**
Refactoring needed

### CODE REVIEW

File: cartridges/int_adyen_SFRA/cartridge/controllers/Adyen.js [hunk 1]
```diff
@@ -195,7 +292,7 @@ function isMethodTypeBlocked(methodType) {
 /**
  * Get OriginKey for Secured Fields
  */
-server.get('GetConfigSecuredFields', server.middleware.https, function (req, res, next) {
+server.get('getConfigurationComponents', server.middleware.https, function (req, res, next) {
```
Comment (Aleffio) [inline cartridges/int_adyen_SFRA/cartridge/controllers/Adyen.js]:
```suggestion
server.get('GetConfigurationComponents', server.middleware.https, function (req, res, next) {
```
Not sure if PascalCase is a requirement for Salesforce,  up to you if you want to change it for consistency

File: cartridges/int_adyen_overlay/cartridge/templates/default/threeds2/adyen3ds2.isml [hunk 2]
```diff
@@ -0,0 +1,114 @@
+<isset name="resultCode" value="${pdict.resultCode}" scope="page"/>
+<isset name="token3ds2" value="${pdict.token3ds2}" scope="page"/>
+<isset name="authorise3DS2" value="${URLUtils.https('Adyen-Authorize3DS2')}" scope="page"/>
+
+<html>
+<head>
+    <script type="text/javascript" src="https://checkoutshopper-test.adyen.com/checkoutshopper/sdk/2.5.0/adyen.js"></script>
```
Comment (Aleffio) [inline cartridges/int_adyen_overlay/cartridge/templates/default/threeds2/adyen3ds2.isml]: Can you use AdyenHelper.getLoadingContext() to get the correct context: TEST/LIVE?

File: cartridges/int_adyen_overlay/cartridge/templates/default/threeds2/adyen3ds2.isml [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+    <script type="text/javascript" src="https://code.jquery.com/jquery-1.7.1.min.js"></script>
+    <link rel="stylesheet"  href="https://checkoutshopper-test.adyen.com/checkoutshopper/sdk/2.5.0/adyen.css"/>
```
Comment (Aleffio) [inline cartridges/int_adyen_overlay/cartridge/templates/default/threeds2/adyen3ds2.isml]: same here, even if it shouldn't matter for the css.

File: cartridges/int_adyen_overlay/cartridge/templates/default/threeds2/adyen3ds2.isml (hunk 2 above)
Comment (maassenbas) [reply cartridges/int_adyen_overlay/cartridge/templates/default/threeds2/adyen3ds2.isml]: Yes, good one.

File: cartridges/int_adyen_SFRA/cartridge/controllers/Adyen.js (hunk 1 above)
Comment (maassenbas) [reply cartridges/int_adyen_SFRA/cartridge/controllers/Adyen.js]: Done

**Pull request merged**